packages/common/lib/types/OpenID4VCIServerMetadata.ts

```diff
@@ -92,50 +17,14 @@ export interface OpenID4VCIServerMetadata {
 export type Oauth2ASWithOID4VCIMetadata = OAuth2ASMetadata & OpenID4VCIServerMetadata;
 
 export interface CredentialIssuer {
-  display?: IssuerDisplay; //OPTIONAL. An array of objects, where each object contains display properties of a Credential issuer for a certain language. Below is a non-exhaustive list of valid parameters that MAY be included:
+  display?: NameAndLocaleExtended; //OPTIONAL. An array of objects, where each object contains display properties of a Credential issuer for a certain language. Below is a non-exhaustive list of valid parameters that MAY be included:
```

Comment:
I find it strange that we now have this interface/type instead of the Display type for all kinds of display properties. It is confusing since the Display type is also there. So either this is incorrect, or suggest to start renaming some.

Something similar happens above for the credentials_supported. The type name is worse than what it was

Reply:
changed this to display, because although the text of spec is only describing `name` and `locale`, the example shows the complete `Display` object. So, I've deleted the `NameAndLocaleExtened` and added the catchall param to `NameAndLocale` itself.
also changed the name of `CredentialIssuerMetadataSupportedCredentials` to `CredentialSupported` to be a bit easier to follow